Content.Server/_Goobstation/StationEvents/Metric/Components/CombatMetricComponent.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Content.Server.StationEvents.Events;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Generic;
+
+namespace Content.Server._Goobstation.StationEvents.Metric.Components;
```

Comment:
Using directive is unnecessary.
```suggestion
using Content.Shared.FixedPoint;

namespace Content.Server._Goobstation.StationEvents.Metric.Components;
```